pages/index.vue

```diff
@@ -96,87 +106,247 @@ const getServiceDescription = (serviceName: string) => {
+              </svg>
+            </button>
+          </div>
+        </div>
+
+        <!-- Mobile Menu Dropdown -->
+        <div
+          v-if="mobileMenuOpen && isAuth0Configured && loggedIn"
+          class="hidden max-[1000px]:block mt-4 p-4 bg-white rounded-lg border border-purple-200 shadow-lg"
+        >
+          <!-- Welcome Message -->
+          <div class="mb-4 pb-4 border-b border-purple-200">
+            <div class="flex items-center space-x-3">
+              <div class="w-10 h-10 rounded-full bg-white border-2 border-green-500 flex items-center justify-center relative">
+                <svg class="w-6 h-6 text-green-500" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M16 7a4 4 0 11-8 0 4 4 0 018 0zM12 14a7 7 0 00-7 7h14a7 7 0 00-7-7z" />
+                </svg>
+                <svg class="absolute -top-1 -right-1 w-4 h-4 text-green-500 bg-white rounded-full" fill="currentColor" viewBox="0 0 20 20">
+                  <path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-9.293a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z" clip-rule="evenodd" />
+                </svg>
+              </div>
+              <div>
+                <p class="text-sm font-semibold text-gray-900">{{ t('auth.welcome', { user: (user as User)?.auth0 || 'User' }) }}</p>
+                <p v-if="(user as User)?.roles?.length" class="text-xs text-gray-500">
+                  {{ (user as User)?.roles?.map(role => translateRoleName(role.name, t)).join(', ') }}
+                </p>
+              </div>
+            </div>
+          </div>
+
+          <!-- User Management (if admin) -->
+          <NuxtLink
+            v-if="isAdmin"
+            to="/admin/users"
+            @click="mobileMenuOpen = false"
+            class="flex items-center space-x-3 px-4 py-3 rounded-lg hover:bg-purple-50 transition-colors mb-2"
+          >
+            <svg class="w-5 h-5 text-gray-600" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M10.325 4.317c.426-1.756 2.924-1.756 3.35 0a1.724 1.724 0 002.573 1.066c1.543-.94 3.31.826 2.37 2.37a1.724 1.724 0 001.065 2.572c1.756.426 1.756 2.924 0 3.35a1.724 1.724 0 00-1.066 2.573c.94 1.543-.826 3.31-2.37 2.37a1.724 1.724 0 00-2.572 1.065c-.426 1.756-2.924 1.756-3.35 0a1.724 1.724 0 00-2.573-1.066c-1.543.94-3.31-.826-2.37-2.37a1.724 1.724 0 00-1.065-2.572c-1.756-.426-1.756-2.924 0-3.35a1.724 1.724 0 001.066-2.573c-.94-1.543.826-3.31 2.37-2.37.996.608 2.296.07 2.572-1.065z" />
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M15 12a3 3 0 11-6 0 3 3 0 016 0z" />
+            </svg>
+            <span class="text-sm text-gray-700">{{ t('auth.userManagement') }}</span>
+          </NuxtLink>
+
+          <!-- Language Picker -->
```

Comment:
Mentioned this on chat, but just for posterity: Language Picker doesn't work in mobile viewport.